framework/components/ecl-forms/ecl-forms-radios/ecl-forms-radios.scss

```diff
@@ -0,0 +1,37 @@
+/*
+ * Radio
+ * @define radio
+ */
+
+.ecl-radio {
+  line-height: 2;
+  margin: 0;
+}
+
+.ecl-radio__label {
+  display: inline-block;
+  padding-left: 34px;
+  position: relative;
+
+  &::before {
+    background: #fff url('../images/svg/radio-unchecked.svg') no-repeat;
+    border: 3px solid transparent;
+    border-radius: 50%;
+    content: '';
+    display: inline-block;
+    height: 1.5em;
+    left: 0;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
```

Comment:
I'm not a big fan of these "transform hack"..
In checkboxes component it has been done using line-height.